Gruntfile.js

```diff
@@ -25,13 +25,18 @@ const ELECTRON_PACKAGE_JSON = 'electron/package.json';
 const PACKAGE_JSON = 'package.json';
 const INFO_JSON = 'info.json';
 
-const LINUX_DESKTOP = {
-  "Version": "1.1",
-  "Name": "Wire",
-  "GenericName": "The most secure collaboration platform",
-  "Categories": "Network;InstantMessaging;Chat;VideoConference",
-  "Keywords": "chat;encrypt;e2e;messenger;videocall",
-  "StartupWMClass": "Wire"
+const LINUX_SETTINGS = {
+  afterInstall: 'bin/deb/after-install.tpl',
+  afterRemove: 'bin/deb/after-remove.tpl',
+  category: 'Network',
+  desktop: {
+    Version: '<%= info.version %>.<%= buildNumber %>',
+    Name: '<%= info.name %>',
+    GenericName: 'The most secure collaboration platform',
```

Comment:
`'<%= info.description %>'`, but we should possibly update the [actual description](https://github.com/wireapp/wire-desktop/blob/master/info.json#L3).

Reply:
/cc @infotexture